matlab/fiff_define_constants.m

```diff
@@ -1,118 +1,63 @@
-function [ FIFF ] = fiff_define_constants()
-%
-%    [ FIFF ] = fiff_define_constants()
-%
-%    Defines structure containing the FIFF constants
-%
-
-%
-%   Author : Matti Hamalainen, MGH Martinos Center
-%   License : BSD 3-clause
-%
-%    Revision 1.23  2009/03/30 11:37:37  msh
-%    Added copying of measurement info blocks from the original like in mne_browse_raw
-%
-%    Revision 1.22  2009/02/16 04:38:21  msh
-%    Fixed comments on coordinate transforms
-%
-%
-%    Revision 1.20  2008/11/16 21:31:23  msh
-%    Added mne_transform_coordinates and new coordinate frame definitions
-%
-%    Revision 1.19  2008/09/30 17:22:39  msh
-%    Added possibility to load unprocessed MaxShield data
-%
-%    Revision 1.18  2008/06/07 21:22:10  msh
-%    Added FIFF_FORWARD_SOLUTION_GRAD constant
-%
-%    Revision 1.17  2008/05/26 10:49:26  msh
-%    Update to incorporate the already weighted lead field basis
-%
-%    Revision 1.16  2008/05/06 20:40:56  msh
-%    Fixed ordering of output for compatibility with maxfilter averager
-%
-%    Revision 1.15  2008/04/16 22:24:57  msh
-%    Added megacq parameters to the measurement info
-%
-%    Revision 1.14  2008/04/16 17:29:28  msh
-%    Added acquisition parameter tags.
-%
-%    Revision 1.13  2008/03/13 19:18:06  msh
-%    Read and write FIFF_MEAS_DATE from/to FIFFB_MEAS_INFO as appropriate
-%
-%    Revision 1.12  2007/01/29 21:21:21  msh
-%    Added reading of the additional source prior covariances.
-%
-%    Revision 1.11  2006/09/25 19:48:16  msh
-%    Added projection item kinds to fiff_define_constants
-%    Changed some fields to int32 in surface structures
-%
-%    Revision 1.10  2006/09/24 18:52:43  msh
-%    Added FIFFV_REF_MEG_CH to fiff_define_constants.
-%    Added coord_frame field to dig point structure.
-%
-%    Revision 1.9  2006/06/22 21:22:46  msh
-%    Take into account the possibility of calibrated compensation matrices
-%
-%    Revision 1.8  2006/05/05 03:50:40  msh
-%    Added routines to compute L2-norm inverse solutions.
-%    Added mne_write_inverse_sol_stc to write them in stc files
-%    Several bug fixes in other files
-%
-%    Revision 1.7  2006/04/28 18:01:27  msh
-%    Added mne_read_bem_surfaces
-%    Fixed errors in mne_read_source_spaces and improved triangle data computation.
-%
-%    Revision 1.6  2006/04/26 19:50:58  msh
-%    Added fiff_read_mri
-%
-%    Revision 1.5  2006/04/23 15:29:39  msh
-%    Added MGH to the copyright
-%
-%    Revision 1.4  2006/04/15 12:21:00  msh
-%    Several small improvements
-%
-%    Revision 1.3  2006/04/10 23:26:53  msh
-%    Added fiff reading routines
-%
-%    Revision 1.2  2005/12/05 20:23:21  msh
-%    Added fiff_save_evoked. Improved error handling.
-%
-%    Revision 1.1  2005/12/05 16:01:04  msh
-%    Added an initial set of fiff writing routines.
-%
+function [FIFF, FWD] = fiff_define_constants()
```

Comment:
Why did you change function signature ?

Reply:
Because it had not been updated during the recent changes, meaning that it was already outdated. I replaced it with the short header from your python function.

We can restore this long history if you think it is useful. And maybe add it to your python function as well, since it was copied from there...